packages/config/tests/api/tests.js

```diff
@@ -470,3 +476,43 @@ test('baseRelDir is true if build.base is overridden', async (t) => {
     .runConfigServer([SITE_INFO_BASE_REL_DIR, FETCH_INTEGRATIONS_EMPTY_RESPONSE])
   t.snapshot(normalizeOutput(output))
 })
+
+test('We call the staging extension API when the apiHost is not api.netlifysdk.com', async (t) => {
+  let baseUrl = ''
+  const returnBaseUrl = (url) => {
+    baseUrl = url
+  }
```

Comment:
seems smelly that we need to do this rather than just intercept the request or something, but 🤷🏼 seems ok